benchmarks/charm++/Makefile

```diff
@@ -5,7 +5,6 @@ DIRS = \
   migrate \
   taskSpawn \
   taskSpawnRecursive \
-  streamingAllToAll \
```

Comment:
Why remove this from the list of benchmarks?

Reply:
It's one of the benchmarks that is currently incompatible with the new API. I need to refactor it, but it breaks test builds if I leave it in(which could mask other issues with the implementation).

Reply:
This should be documented so it can be brought back later.